core/src/main/scala/caliban/schema/Schema.scala

```diff
@@ -471,6 +471,8 @@ trait TemporalSchema {
       PureStep(format(value))
   }
 
+  lazy val sampleDate: ZonedDateTime = ZonedDateTime.ofInstant(Instant.EPOCH, ZoneId.systemDefault())
```

Comment:
I think we should use the `ZoneOffset.UTC` instead of `systemDefault()` here since it may result in different behaviors in two schemas generated on different computers.

Reply:
Good point, I changed.